.github/workflows/api-e2e-mssql-multitenant.yml

```diff
@@ -43,6 +43,9 @@ jobs:
       - name: Copy admin console folder to docker context
         run: cp -r ../EdFi.Ods.AdminApi.AdminConsole ../../Docker/Application
 
+      - name: Copy admin api common folder to docker context
+        run: cp -r ../EdFi.Ods.AdminApi.Common ../../Docker/Application
```

Comment:
This is fine for a short term solution, but you don't need it anymore. The Dockerfile now has a capability for introducing context in a different directory. NO NEED TO CHANGE IN THIS PULL REQUEST 😄 .

Take a look at the [DMS dockerfile](https://github.com/Ed-Fi-Alliance-OSS/Data-Management-Service/blob/main/src/dms/Dockerfile), lines 18-20. They allow copying files from a different context, other than the current directory.

You provide the additional context at the command line:

```powershell
&docker buildx build -t $dockerTagDMS -f Dockerfile . --build-context parentdir=../
```

Or in a docker-compose file:

```yaml
services:
  dms:
    build:
      context: ../../src/dms/
      additional_contexts:
        parentdir: ../../src/
```